.buildkite/pipeline.trigger.compliance.tests.sh

```diff
@@ -26,7 +26,10 @@ steps:
 EOF
 
 # Generate each test we want to do.
-compliance_test 8.14.0-SNAPSHOT 3.1.5
+## Requires changes in elastic-package to be able to test log_synthetic_mode test package
+# compliance_test 8.15.0-SNAPSHOT 3.2.0
```

Comment:
This could be enabled once a new elastic-package version is released with this change
https://github.com/elastic/elastic-package/pull/1914